### DISCUSSION TITLE
Feature Request: Belkin Wemo Doorbell through scrypted

### DISCUSSION THREAD

**Discussion by yonigold1** [koush/scrypted#451] [Ideas]
Just wanted to start off by saying how much I love scrypted I have it setup with my yi-hacked cameras and it works flawlessly! 😃

So I have a Belkin Wemo HomeKit Doorbell and I was thinking about pairing it to my Home Assistant setup instead of directly to HomeKit (So I can use it on Alexa and Google). I thought about all the features that scrypted has and realized that adding it to Home Assistant would only get the video and audio. 

Would it be possible to create a plugin that can control already-homekit enabled cameras?

**Comment by yonigold1** [comment]
Would this be possible to do?

**Reply by simplenotezy** [reply]
Can you elaborate a bit on which features you'd expect / want?
